### ISSUE TITLE
Gnome 3 App Icon not showing

### ISSUE THREAD

**Issue by ohaessler** [RocketChat/Rocket.Chat.Electron#430]
On RHEL 7, the App icon is not showing when starting the App.

- Hit the "Windows" Key to get the app overview
- Type "rocket"

The App is shown without a icon. The log file shows the following error:

gnome-session[3768]: (org.gnome.Software:4328): GsPlugin-WARNING **: failed to load stock icon rocketchat: Icon 'rocketchat' not present in theme

**Comment by Rumbles** [comment]
This issue has been fixed in the past but seems to have reappeared in 2.7

**Comment by Rumbles** [comment]
I'm using Fedora, so slightly different, I don't see the gnome-session error you quote. I remember in the past this issue was fixed as a colleague reported it. But I have just installed 2.7 from the releases and the issue has reappeared.

**Comment by bagasse** [comment]
Hi, same problem here (on CentOS 7)
The /usr/share/icons/hicolor/512x512/apps/rocketchat-desktop.png file is not used by gnome.
I've solved this issue on my desktop via the following steps: 

1. Downloading an svg version of the icon
2. Put this file in `/usr/share/icons/hicolor/scalable/apps/rocketchat-desktop.svg`
3. Force gtk icon cache for this theme with the `sudo gtk-update-icon-cache /usr/share/icons/hicolor` command

To solve this globaly, the RPM have to
1. provides an svg version of the icon and place it like above
2. force the refresh of icon cache in the %post section for the RPM spec file (if the command exist). For this point, i don't know how this is handled by other desktop envs (kde...)

I didn't find any RPM spec file in this repo so i didn't add a pull request for this.

**Comment by Rumbles** [comment]
Mine actually started working again on fedora after a reboot (not sure if that fixed it but the icon works again for me now)

**Comment by gigawhitlocks** [comment]
I'm seeing this in both MATE and xfce4 on Arch

**Comment by ecwpz91** [comment]
Base on [bagasse](https://github.com/RocketChat/Rocket.Chat.Electron/issues/430#issuecomment-303436818)'s comment:

```sh
sudo curl -o /usr/share/icons/hicolor/scalable/apps/rocketchat.svg \
https://raw.githubusercontent.com/RocketChat/Rocket.Chat.Artwork/master/Logos/icon.svg

sudo gtk-update-icon-cache /usr/share/icons/hicolor
```

Reboot / logout to restart Gnome.

**Comment by bagasse** [comment]
This is fixed for me after update from 2.11.0 to 2.13.2. Thanks.

**Comment by tassoevan** [comment]
I'm closing it due to inactivity.
